HSRPizzaHelper/View/SysViews/AboutView.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  AboutView.swift
+//  HSRPizzaHelper
+//
+//  Created by Bill Haku on 2023/5/5.
+//
+
+import SwiftUI
+
+struct AboutView: View {
+    let appVersion = Bundle.main
+        .infoDictionary!["CFBundleShortVersionString"] as! String
+    let buildVersion = Bundle.main.infoDictionary!["CFBundleVersion"] as! String
```

Comment:
avoid force unwrap

Reply:
or use // swiftlint:disable ... to avoid warning

Reply:
how to deal with it? convert `Any?` to `String`

Reply:
Just `(xx as? String) ?? "")´